cli/README.md

```diff
@@ -6,22 +6,32 @@ EMAP star database and the PACS image system. Once a set of queues are
 populated the consumers can be started, updated and the system extractions
 stopped cleanly.
 
+## Prerequisites
+
+`PIXL CLI` requires Python version 3.10.
```

Comment:
Feels like this is defined in the pyproject.toml already but also useful to know 🤷

Reply:
Yeah, it's nice to have it here, because new developer might not necessary look at the `pyproject.toml`.
In fact, it probably makes sense to mention this in the top-level README as well, maybe some instructions to create a conda/virtual environment with the correct Python version